src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/HistoriskeVurderinger/HistoriskeVurderinger.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+import { useNavigate } from 'react-router-dom';
+
+import { useBehandling } from '../../../../../context/BehandlingContext';
+import { ToggleName } from '../../../../../context/toggles';
+import { useToggles } from '../../../../../context/TogglesContext';
+import { IBehandling } from '../../../../../typer/behandling';
+import { IFagsak } from '../../../../../typer/fagsak';
+import { BehandlingsMenyButton } from '../../../../Felleskomponenter/Flytelementer';
+interface IProps {
+    behandling: IBehandling;
+    fagsak: IFagsak;
+    onListElementClick: () => void;
+}
+
+const HistoriskeVurderinger: React.FC<IProps> = ({ behandling, fagsak, onListElementClick }) => {
+    const { behandlingILesemodus } = useBehandling();
+    const navigate = useNavigate();
+    const { toggles } = useToggles();
+    return (
+        toggles[ToggleName.seHistoriskeVurderinger] &&
+        behandling.kanEndres &&
+        !behandlingILesemodus && (
+            <>
+                <BehandlingsMenyButton
+                    variant="tertiary"
```

Comment:
Knapp i behandlingsmenyen for å kunne få oversikt over tidligere vurderinger av behandlingen. 
Tar gjerne input på "navngivning" på både knappeinnhold og generelt innhold